src/main/java/alerters/EmailNotifier.java

```diff
@@ -0,0 +1,14 @@
+package alerters;
+
+import batteryStates.BreachType;
+
+public class EmailNotifier implements IBreachObserver{
+
```

Comment:
Partial Implementation of EmailNotifier .. go through the problem statement again